frontend-react/src/pages/submissions/SubmissionTable.test.tsx

```diff
@@ -1,31 +1,14 @@
-import { Fixture, MockResolver } from "@rest-hooks/test";
+import { Fixture } from "@rest-hooks/test";
 import { screen, within } from "@testing-library/react";
-import { ReactElement } from "react";
-import { CacheProvider } from "rest-hooks";
 
 import SubmissionsResource from "../../resources/SubmissionsResource";
-import { renderWithRouter } from "../../utils/CustomRenderUtils";
+import { renderApp } from "../../utils/CustomRenderUtils";
 import { mockSessionContext } from "../../contexts/__mocks__/SessionContext";
 import { MemberType } from "../../hooks/UseOktaMemberships";
 import { Organizations } from "../../hooks/UseAdminSafeOrganizationName";
 
 import SubmissionTable from "./SubmissionTable";
 
-// const { addFeatureFlag, removeFeatureFlag } = _exportForTesting;
-
-// TODO: Move this to CustomRenderUtils.tsx once we stop mocking rest-hooks.
```

Comment:
🔥 🔥 🔥

Reply:
This was the catalyst for going forward with the consolidation, since even though we're not done with rest-hooks yet the changes required to tests still using them to use fixtures was minimal 🙂.